Ports/harfbuzz/package.sh

```diff
@@ -14,6 +14,8 @@ configopts=(
 )
 
 configure() {
+    # Some ports require -fPIC for static linked harfbuzz
```

Comment:
Why is this needed?

Reply:
> Why is this needed?

Because harfbuzz is built as a static binary.

Should I use `-DBUILD_SHARED_LIBS=ON` instead? `-fPIC` is unnecessary if harfbuzz is built as a shared object.

Reply:
I mean: Why does cairo require PIC code in a static library? I.e. what goes wrong if you don't add `-fPIC` here?

Reply:
Sorry, I noticed that it's not for cairo but for pango. I'll revert this change.

You will get the "recompile with -fPIC" errors when compiling pango.
```
.../libharfbuzz.a(harfbuzz.cc.o): relocation R_X86_64_PC32 against symbol `_ZN11hb_buffer_t19_cluster_group_funcERK15hb_glyph_info_tS2_' can not be used when making a shared object; recompile with -fPIC
```